src/components/SideBar/SideBarHeader.tsx

```diff
@@ -0,0 +1,62 @@
+import Link from 'next/link'
+import { useEffect, useRef, useState } from 'react'
+
+const SideBarHeader: React.FC<{ header: string }> = ({ header }) => {
+  const [isProfileActive, setIsProfileActive] = useState(false)
+
+  const profileRef = useRef<HTMLDivElement | null>(null)
+
+  useEffect(() => {
+    const handleProfile = (e: MouseEvent) => {
```

Comment:
Maybe name this as `handleClickOutside` or something if this is used to close profile when clicked outside.